docs/adr/0008-split-bindings-to-modules-automatically.md

```diff
@@ -0,0 +1,64 @@
+# [short title of solved problem and solution]
+
+* Status: [proposed superseded by [gogo](0008-split-bindings-to-modules-automatically.md)]
+* Deciders: []
+* Date: [2022-07-11 when the decision was last updated]
+
+## Context and Problem Statement
+
+In his current state, uniffi only allow to import any generated code to a single file. This is embarassing because in languages used to make bindings, the user has to import submodules.
```

Comment:
Please add more about your use case here.  How is you hand-written code structured and how do you want the generated module(s) to fit in to that?

Also can you add give describe the drawbacks in a more precise way than "embarrassing".  Is it a time sink in some way?  Does it limit what UniFFI can do?

Reply:
I pushed once again. Fell free to get a look.